src/partials/module.html

```diff
@@ -36,10 +36,13 @@
           <td>
             {{ ctrl.getDatasource(panel) }}
           </td>
-          <td>
-            <button class="btn gf-form-btn btn-primary" type="button" ng-click="ctrl.showExportModal(panel.id, target)">
-              Export
-            </button>
+          <td style="display: inline-block" bs-tooltip="!panel.datasource ? 'Datasource is not supported' : null">
```

Comment:
let's add some class instead of using inline styles

Reply:
please, avoid implicity: `!panel.datasource` -> `panel.datasource === undefined`